test_altimeter.ino

```diff
@@ -0,0 +1,45 @@
+#if false
+
+#include "altimeter.h"
+
+Altimeter alt;
+
+void setup() {
+  Serial.begin(9600);
+  while (!Serial) { /* Wait for Serial monitor to connect */ }
+  Serial.println("Connecting to sensor...");
+  alt.initialize();
+}
+
+void loop() {
+  // Exponential backoff: each time it waits longer than the last.
+  // That way, if a wire is loose or missing, it doesn't flood the console with messages.
+  static unsigned int backoff = 100;
+
+  if (alt.getStatus() != Altimeter::ACTIVE) {
+    Serial.print("Not connected. Trying again in ");
+    Serial.print(backoff);
+    Serial.println("ms...");
+    delay(backoff);
+    backoff *= 2;
+    alt.initialize();
+  } else {
+    float altitude;
+    setSeaLevel(altitude);
```

Comment:
I don't think this will work. You're saying there's a float value called `altitude` but not saying what the value is, and then setting the sea level to that indeterminate value. I was thinking of putting something like `const float SEA_LEVEL_PRESSURE = 1013.25F; setSeaLevel(SEA_LEVEL_PRESSURE);` after the `alt.initialize()` calls.

Reply:
agreed